packages/vaex-core/setup.py

```diff
@@ -2,7 +2,7 @@
 import sys
 import os
 import imp
-from setuptools import Extension
+from pybind11.setup_helpers import Pybind11Extension
```

Comment:
still need the import 
```suggestion
from setuptools import Extension
from pybind11.setup_helpers import Pybind11Extension
```